### PR TITLE
up app

### PR DESCRIPTION
**Pull request by PhongTaKiai**
- close: #5 
Simulator Screenshot - iPhone 15 Pro - 2024-02-15 at 15 57 39

### CODE REVIEW

**Review by rick1919 — COMMENTED** [review]
The UI is LGTM!

File: lib/src/features/home/presentation/home_page.dart [hunk 1]
```diff
@@ -125,7 +125,9 @@ class HomePage extends ConsumerWidget {
         ),
       ),
       floatingActionButton: IconButton(
-        onPressed: () {},
+        onPressed: () {
+          context.push('/up-app');
```
Comment (mafreud) [inline lib/src/features/home/presentation/home_page.dart:129]: Upload-app is a better name.

File: lib/src/features/up_app/up_app_page.dart [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import 'package:app_buy_sell/src/constants/color_constant.dart';
+import 'package:flutter/material.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+
+class UpAppPage extends HookConsumerWidget {
```
Comment (mafreud) [inline lib/src/features/up_app/up_app_page.dart:5]: You don't use ```WidgetRef``` or ```hooks```.  ```Stateless Widget``` is enough.

**Review by mafreud — CHANGES_REQUESTED** [review]

**Comment by PhongTaKiai** [comment]
@mafreud fixed, thank

**Pull request merged**